app/js/createUserObject.js

```diff
@@ -1,71 +1,114 @@
 const numberOfQuestions = 30
 
 /**
- * gets all users score results from api and add percentage values
- *
- * @return Array - containing the score results with percentage
+ * Get all the test results from the API.
  */
-async function applyPercent() {
-    let scoreArr = await fetch("http://localhost:8080/result", {method: 'get'})
-        .then(function (data) {
-            return data.json()
-        })
-
-    scoreArr.data.forEach(function (score) {
-        score.percent = (score.score / numberOfQuestions * 100).toFixed(2)
+async function getResults() {
+    let resultsArr = await fetch("http://localhost:8080/result", {method: 'get'})
+    .then(function (data) {
+        return data.json()
     })
-    return scoreArr.data
+    return resultsArr.data
 }
 
 /**
- * gets all users name and email from API
- *
- * @return Array - containing the user info (Name and Email)
+ * Gets users from the API.
+ * Filters out users that have been soft-deleted from the database.
+ * @return Array of user objects
  */
-async function getNameAndEmail() {
+async function getUsers() {
     let users = await fetch("http://localhost:8080/user", {method: 'get'})
         .then(function (data) {
             return data.json()
         })
+    let filteredUsersArray = users.data.filter( function(value, index, arr) {
```

Comment:
just a question, what are we "filtering" here? because I don't see any filter condition inside the function.